src/services/importing/conflicting-plugins-service.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Yoast\WP\SEO\Services\Importing;
+
+use Yoast\WP\SEO\Config\Conflicting_Plugins;
+
+/**
+ * Detects plugin conflicts.
+ */
+class Conflicting_Plugins_Service {
+
+	/**
+	 * Detects the conflicting plugins.
+	 *
+	 * @return array A list of all active conflicting plugins.
+	 */
+	public function detect_conflicting_plugins() {
+		$all_active_plugins = $this->get_active_plugins();
+
+		// Search for active plugins.
+		return $this->get_active_conflicting_plugins( $all_active_plugins );
+	}
+
+	/**
+	 * Deactivates the specified plugin(s) if any, or the entire list of known conflicting plugins.
+	 *
+	 * @param string|array|false $plugins (optional) The plugin filename, or array of plugin filenames, to deactivate.
+	 */
+	public function deactivate_conflicting_plugins( $plugins = false ) {
+		// If no plugins are specified, deactivate any known conflicting plugins that are active.
+		if ( ! $plugins ) {
+			$plugins = $this->detect_conflicting_plugins();
+		}
+
+		// In case of a single plugin, wrap it in an array.
+		if ( \is_string( $plugins ) ) {
+			$plugins = [ $plugins ];
+		}
+
+		if ( ! is_array( $plugins ) ) {
+			return;
+		}
+
+		// Deactivate all specified plugins across the network, while retaining their deactivation hook.
+		\deactivate_plugins( $plugins, false, true );
```

Comment:
When there are no items in `$plugins`, a fatal is thrown stating that the `deactivate_plugins()` function is undefined:

```
Fatal error: Uncaught Error: Call to undefined function deactivate_plugins() in /var/www/html/wp-content/plugins/wordpress-seo-git/src/services/importing/conflicting-plugins-service.php: 45
Stack trace:
#0 /var/www/html/wp-content/plugins/wordpress-seo-git/src/actions/importing/deactivate-conflicting-plugins-action.php(78): Yoast\WP\SEO\Services\Importing\Conflicting_Plugins_Service->deactivate_conflicting_plugins(Array)
#1 /var/www/html/wp-content/plugins/wordpress-seo-git/src/routes/importing-route.php(89): Yoast\WP\SEO\Actions\Importing\Deactivate_Conflicting_Plugins_Action->index()
#2 /var/www/html/wp-includes/rest-api/class-wp-rest-server.php(1140): Yoast\WP\SEO\Routes\Importing_Route->execute(Object(WP_REST_Request))
#3 /var/www/html/wp-includes/rest-api/class-wp-rest-server.php(987): WP_REST_Server->respond_to_request(Object(WP_REST_Request), '/yoast/v1/impor...', Array, NULL)
#4 /var/www/html/wp-includes/rest-api/class-wp-rest-server.php(414): WP_REST_Server->dispatch(Object(WP_REST_Request))
#5 /var/www/html/wp-includes/rest-api.php(370): WP_REST_Server->serve_request('/yoast/v1/impor...')
#6 /var/www/html/wp-includes/class-wp-hook.php(303): rest_api_loaded(Object(WP))
#7 /var/www/html/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#8 /var/www/html/wp-includes/plugin.php(518): WP_Hook->do_action(Array)
#9 /var/www/html/wp-includes/class-wp.php(388): do_action_ref_array('parse_request', Array)
#10 /var/www/html/wp-includes/class-wp.php(750): WP->parse_request('')
#11 /var/www/html/wp-includes/functions.php(1291): WP->main('')
#12 /var/www/html/wp-blog-header.php(16): wp()
#13 /var/www/html/index.php(17): require('/var/www/html/w...')
#14 {main
}
  thrown in /var/www/html/wp-content/plugins/wordpress-seo-git/src/services/importing/conflicting-plugins-service.php on line 45
```